image.go

```diff
@@ -7,12 +7,42 @@ import (
 	"time"
 
 	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/types"
 )
 
+type EditableImage interface {
```

Comment:
Nice organization :) I'm curious to know why the other SetXXX methods (e.g., SetCmd) aren't included. And are methods like CreatedAt and Entrypoint not also "getters"?

Reply:
The intention behind `EditableImage` interface is to create a type that has Setters an Getters that we need to annotate an image using `pack manifest` command
And [Here](https://github.com/buildpacks/imgutil/blob/6eb1a7027489a6508012fd461b27a5bd065e3780/options.go#L112) we can actually add a local/layout image that should have these set of methods must be defined, and for [MultiArch builder](https://github.com/buildpacks/pack/blob/a78dd269bb8bd5bf566fca7ebba72120430baee0/pkg/buildpack/builder.go#L86) I am using this interface to mutate image accordingly